.github/actions/private-tangle/setup/action.yml

```diff
@@ -0,0 +1,41 @@
+name: 'private-tangle-setup'
+description: 'Setup a private tangle'
+runs:
+  using: "composite"
+  steps:
+    - name: Setup private tangle
+      shell: bash
+      run: |
+        # TODO: use next lines when a working hornet release is published
+        # # Download the private_tangle setup from the hornet repo.
+        # mkdir private_tangle
+        # cd private_tangle
+        # # Use the output of https://api.github.com/repos/iotaledger/hornet/releases/latest once there's a 2.0 Hornet release.
+        # DOWNLOAD_URL=$(curl "https://api.github.com/repos/iotaledger/hornet/releases" | jq -r '.[0].assets[] | select(.name | contains("private_tangle")) | .browser_download_url')
+        # echo "Downloading private tangle from $DOWNLOAD_URL"
+        # curl -L -o private_tangle.tar.gz $DOWNLOAD_URL
+        # tar -xf private_tangle.tar.gz
+        # TODO: remove next lines when a working hornet release is published
+        git clone https://github.com/iotaledger/hornet.git
+        cd hornet/private_tangle
+        # Set minPoWScore = 1 since the default (0) doesn't work with wasm_miner.rs in iota.rs currently.
```

Comment:
This has been fixed?

Reply:
Yes, just copied the whole workflow from identity.rs, will update